temel_bilgiler/kurulumlar/bilgisayar_temelleri.md

```diff
@@ -0,0 +1,44 @@
+### Giriş
+
+Söylemeye gerek yok, kodlamayı öğrenirken zamanınızın çoğunu bilgisayar başında geçireceksiniz. Bu nedenle, bilgisayarınızla nasıl çalışacağınızı anlamak çok önemlidir. Bu derste, sizi hızlı bir şekilde bilgisayar temelleri konusunda bilgilendirecek bir kaynak tanıtacağız.
+
+### Derse Genel Bakış
+
+Bu bölüm, bu derste öğreneceğiniz konulara genel bir bakış içerir.
```

Comment:
This line too.
fix: Bu bölüm, bu derste öğreneceğiniz konuların genel bir özetini içerir.